app/routables/index/styles.scss

```diff
@@ -1,168 +1,307 @@
-            position: absolute;
-            background: #ffdb5b;
-            color: #353535;
-            top: 20px;
-            right: 20px;
-            padding: 10px 16px;
-            font-size: 22px;
-            margin: 0;
-          }
+      .toggler:checked + .hamburger {
+        div {
+          transform: rotate(135deg);
         }
-        .text {
-          padding: 28px;
-          position: relative;
-          background-color: #272727;
-          margin-bottom: 40px;
-          h4 {
-            font-family: $bold;
-            font-size: 30px;
-            color: white;
-            small {
-              font-family: $light;
-              font-size: 24px;
-              margin-left: 10px;
-            }
-          }
-          p,
-          dt,
-          dd {
-            color: white;
-            line-height: 180%;
-            font-size: 16px;
-          }
-          dl {
-            margin: 22px 0;
-            display: flex;
-            flex-flow: wrap;
-            dt {
-              width: 40%;
-            }
-            dd {
-              width: 60%;
-              margin: 0; // to overide the browser default.
-            }
-          }
-          p {
-            a {
-              color: white;
-              text-decoration: underline;
-              &:hover {
-                text-decoration: none;
-                color: #ffdb5b;
-              }
-            }
+        div::before,
+        div::after {
+          top: 0;
+          transform: rotate(90deg);
+        }
+        .toggler:checked ~ .menu {
+          visibility: visible;
+          div {
+            transform: scale(1);
+            transition-duration: 0.8s;
           }
         }
       }
-    }
-  }
-}
+      .menu {
+        position: fixed;
+        top: 0px;
+        right: 0px;
+        width: 100%;
+        height: 100%;
+        visibility: hidden;
+        overflow: hidden;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      .one {
+        background: #040009;
+        border-radius: 50%;
+        width: 200vw;
+        height: 200vw;
+        display: flex;
+        flex: none;
+        align-items: center;
+        justify-content: center;
+        transform: scale(0);
+        transition: all 0.4s ease;
+      }
+      .two {
+        text-align: center;
+        max-width: 90vw;
+        max-height: 100vh;
+        opacity: 0;
+        transition: opacity 0.4s ease;
+      }
+      ul {
+        li {
+          list-style: none;
+          color: black;
+          font-size: 45px;
+          padding: 40px;
 
-section#contact {
-  background: #252525;
-  padding: 60px;
-  div {
-    text-align: center;
-    margin: 25px;
-    p {
-      color: white;
-      font-size: 20px;
-      line-height: 150%;
-      a {
-        color: white;
+          a {
+            text-decoration: none;
+            transition: color 0.4s ease;
+            color: #fff;
```

Comment:
Consider simply typing `white` rather than `#fff`. It's one of the named CSS colours. Reads easier.